lib/stack_master/parameter_resolvers/one_password.rb

```diff
@@ -0,0 +1,48 @@
+module StackMaster
+  module ParameterResolvers
+    class OnePassword < Resolver
+      OnePasswordNotFound = Class.new(StandardError)
+
+      array_resolver
+
+      def initialize(config, stack_definition)
+        @config = config
+        @stack_definition = stack_definition
+      end
+
+      def resolve(params={})
+        raise RuntimeError, "1password requires the `OP_SESSION_<name>` to be set" if ENV.keys.grep(/OP_SESSION_\w+$/).empty?    
+        get_items(params)
+      end
+
+      private
+
+      def op_get_item(item, vault)
+        begin
+          %x(op get item --vault='#{vault}' '#{item}')
+        rescue Errno::ENOENT
+          raise RuntimeError, "The op cli needs to be installed in the PATH"
+        rescue => exception
```

Comment:
Does this throw a sensible error message when and item isn't present in the vault?

Reply:
does now :)